plugin/evm/validators_default.go

```diff
@@ -52,19 +54,37 @@ var songbirdNodeIDs = []string{
 
 var flareNodeIDs = []string{}
 
-// getDefaultValidators gets the set of default validators, with their respective
-// weights, as defined in the legacy code base of Flare.
-func getDefaultValidators(chainID *big.Int) (map[ids.ShortID]uint64, error) {
+type ValidatorsDefault struct {
+	validators map[ids.ShortID]uint64
+	steps      []Step
+}
+
+type Step struct {
+	Epoch  uint64
+	Cutoff int
+}
+
+func NewValidatorsDefault(chainID *big.Int) (*ValidatorsDefault, error) {
 
 	var weight uint64
 	var nodeIDs []string
+	var steps []Step
 	switch {
 	case chainID.Cmp(params.CostonChainID) == 0:
 		nodeIDs = costonNodeIDs
 		weight = costonValidatorWeight
+		steps = []Step{
```

Comment:
Could you please put names for these epochs in the comments?

Reply:
I don't know what you mean?